ui/app/src/components/GlobalApp/GlobalApp.tsx

```diff
@@ -52,11 +60,79 @@ interface GlobalAppProps {
   footerHtml: string;
 }
 
+export function GlobalAppRouterProvider(props: GlobalAppProps) {
+  const { passwordRequirementsMinComplexity } = props;
+  const globalNavigation = useGlobalNavigation();
+  const { classes } = useStyles();
+
+  const router = createHashRouter(
+    createRoutesFromElements(
+      <Route path="/" element={<GlobalApp {...props} />}>
+        <Route path="/sites" element={<SiteManagement />} />
+        <Route
+          path="/users"
+          element={<UserManagement passwordRequirementsMinComplexity={passwordRequirementsMinComplexity} />}
+        />
+        <Route path="/groups" element={<GroupManagement />} />
```

Comment:
Is there a way of defining the routes without the instantiation of the component at the definition?